### DISCUSSION TITLE
Can you provide a full site template use daisyui?

### DISCUSSION THREAD

**Discussion by skyboy2020** [saadeghi/daisyui#725] [Ideas / feature requests]
Without the site-wide template, the site-wide layout and design cannot be done well. Can you provide a site-wide template that uses components for reference? like this:  https://themes.getbootstrap.com/

**Comment by saadeghi** [comment]
I don't agree about "cannot be done well"  
But sure, I will create some full-page templates as soon as I get some free time.

I'm also moving this to Discussions because it's not an issue
